recipe/meta.yaml

```
@@ -35,7 +34,11 @@ requirements:
     - tenacity >=8.1.0,<9.0.0
     - aiohttp >=3.8.3,<4.0.0
     - async-timeout >=4.0.0,<5.0.0
```

```suggestion
    - async-timeout >=4.0.0,<5.0.0  # [py<311]
```

https://github.com/hwchase17/langchain/blob/v0.0.157/pyproject.toml#L65